app/templates/contact.html

```diff
@@ -5,13 +5,26 @@
         <div class="contact-image">
             <img src="./static/img/logo.jpg">
         </div>
-        <div>
+        <div class="contact-form">
             <h1>{{ title }}</h1>
-            <p>Want to get in touch? Send me a message at <a href="mailto:not_a_real@email.com">not_a_real@email.com</a>!</p>
+            <p>Want to get in touch? Send me a message: </p>
+            <form id="contact-form" action="{{ url }}/send-email" method="POST"> 
```

Comment:
I think it might be safer to use url_for('send_email') so that it doesn't break if we change the route